ee/agent/internal/informer_modules.go

```diff
@@ -0,0 +1,86 @@
+package internal
+
+import (
+	"reflect"
+
+	"github.com/formancehq/stack/components/agent/internal/generated"
+	"github.com/formancehq/stack/libs/go-libs/logging"
+	"google.golang.org/protobuf/types/known/structpb"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
+)
+
+func getStatus(logger logging.Logger, unstructuredModule *unstructured.Unstructured) *structpb.Struct {
+	status, found, err := unstructured.NestedMap(unstructuredModule.Object, "status")
```

Comment:
After more thinking about this, I think we should only deal with conditions.
The "status" field can change, and it changes regularly, sending it fully will bloat membership with a ton of different status properties.

Reply:
I can wraped the status in a known struct like ModuleStatusWithConditions, which has `Info`, `Ready`, and `[]Conditions`.
We might need the `generation` field to guess what's the latest condition ?

Reply:
yes, ready, info and conditions should be ok.
generation is already included in conditions.